packages/components/src/spectrum/View.tsx

```diff
@@ -23,13 +30,46 @@ export type ViewProps = Omit<SpectrumViewProps<6>, 'backgroundColor'> & {
 
```

Comment:
Can you delete this blank line so the comment is actually with the function? Also the last blank line of the comment... and check that comment for typos ("paletee" should be "palette"